pkg/controller/nstemplateset/nstemplateset_controller.go

```diff
@@ -71,10 +71,19 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 	if err := c.Watch(&source.Kind{Type: &corev1.Namespace{}}, commoncontroller.MapToOwnerByLabel("", toolchainv1alpha1.OwnerLabelKey)); err != nil {
 		return err
 	}
+
+	// watch for all cluster resource kinds associated with an NSTemplateSet
 	for _, clusterResource := range clusterResourceKinds {
-		// watch for all cluster resource kinds associated with an NSTemplateSet
-		if err := c.Watch(&source.Kind{Type: clusterResource.objectType}, commoncontroller.MapToOwnerByLabel("", toolchainv1alpha1.OwnerLabelKey)); err != nil {
-			return err
+		switch clusterResource.gvk.Kind {
+		case "ClusterResourceQuota":
```

Comment:
Let's create a const for `ClusterResourceQuota` and reuse it here and in https://github.com/codeready-toolchain/member-operator/blob/eb709dea0937d7645ba9341a42f29ec8c1439e28/pkg/controller/nstemplateset/cluster_resources.go#L55

Reply:
I guess this is not needed now that we removed the switch statement

Reply:
Yep